tests/paddle/test_paddle_autolog.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+import paddle
+import paddle.nn as nn
+import paddle.vision.transforms as T
+from paddle.static import InputSpec
+import mlflow
+import mlflow.paddle
+from mlflow.utils.file_utils import TempDir
+from mlflow.paddle._paddle_autolog import _get_optimizer_name
+
+NUM_EPOCHS = 2
+
+pytestmark = pytest.mark.large
+
+
+@pytest.fixture
+def paddle_model():
+    device = paddle.set_device("cpu")  # or 'gpu'
+
+    net = nn.Sequential(nn.Flatten(1), nn.Linear(784, 200), nn.Tanh(), nn.Linear(200, 10))
```

Comment:
Can we use a smaller dataset? It takes a long time to traing a model with the MNIST dataset.

https://github.com/mlflow/mlflow/pull/4751/checks?check_run_id=3446916341#step:6:444

```python
=========================== slowest 5 test durations ===========================
47.78s call     tests/paddle/test_paddle_autolog.py::test_paddle_autolog_log_models_configuration[True]
47.17s call     tests/paddle/test_paddle_model_export.py::test_pyfunc_serve_and_score
31.01s setup    tests/paddle/test_paddle_autolog.py::test_paddle_early_stop_params_logged[2]
30.14s call     tests/paddle/test_paddle_autolog.py::test_paddle_autolog_persists_manually_created_run
29.87s setup    tests/paddle/test_paddle_autolog.py::test_paddle_autolog_non_early_stop_callback_does_not_log
=============================== warnings summary ===============================
```

Reply:
Sure!